opal/static/js/opal/services/fields_translater.js

```diff
@@ -62,17 +62,31 @@ angular.module('opal.services').service('FieldTranslater', function($rootScope){
     return subrecord;
   }
 
+  this.cleanEmptyStrings = function(fieldValue){
```

Comment:
I expect a thing called `cleanEmptyStrings` to conceptually be able to handle multiple strings - by clean you mean trim?